node/parachain/src/chain_spec.rs

```diff
@@ -40,12 +40,14 @@ pub struct Extensions {
 
 impl Extensions {
 	/// Try to get the extension from the given `ChainSpec`.
+	#[allow(clippy::borrowed_box)]
 	pub fn try_get(chain_spec: &Box<dyn sc_service::ChainSpec>) -> Option<&Self> {
```

Comment:
```suggestion
	#[allow(clippy::borrowed_box)]
	pub fn try_get(chain_spec: &dyn sc_service::ChainSpec) -> Option<&Self> {
```

And then when it is called in in `command.rs`, call it like this:

```rust
let extension = chain_spec::Extensions::try_get(&*config.chain_spec);
```

Reply:
I suggested the same change upstream https://github.com/paritytech/cumulus/pull/272

Reply:
Nice. Some of my hesitation with `clippy` in general is keeping code that's copied from upstream in its original form so that subsequent merges from upstream aren't difficult.

Reply:
Addressed in bc65b15

Reply:
> ...code that's copied from upstream...

Yeah I agree. I think in general we should make an effort to contribute back upstream, and when it isn't feasible we can disable the lint. I like that you disabled the lints in our hotfixes which are mostly copied from Frontier.